client/my-sites/themes/themes-site-selector-modal.jsx

```diff
@@ -83,12 +83,12 @@ const ThemesSiteSelectorModal = React.createClass( {
 	 * but only if it also has a header, because the latter indicates it really needs
 	 * a site to be selected and doesn't work otherwise.
 	 */
-	wrapOption( option ) {
+	wrapOption( option, name ) {
```

Comment:
There are two occurrences in this component of `wrapOption` being called with one argument. Should we fix them to provide a `name`? Or have the method default to some value for undefined `name`?